sdk/core/core-http/src/browserFetchHttpClient.ts

```diff
@@ -1,26 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-import {
-  FetchHttpClient,
-  CommonRequestInfo,
-  CommonResponse,
-  CommonRequestInit
-} from "./fetchHttpClient";
-import { HttpOperationResponse } from "./httpOperationResponse";
-import { WebResourceLike } from "./webResource";
-
-export class BrowserFetchHttpClient extends FetchHttpClient {
-  prepareRequest(_httpRequest: WebResourceLike): Promise<Partial<RequestInit>> {
-    return Promise.resolve({});
-  }
-
-  processRequest(_operationResponse: HttpOperationResponse): Promise<void> {
-    return Promise.resolve();
-  }
-
-  // eslint-disable-next-line @azure/azure-sdk/ts-apisurface-standardized-verbs
-  fetch(input: CommonRequestInfo, init?: CommonRequestInit): Promise<CommonResponse> {
-    return fetch(input, init);
-  }
-}
```

Comment:
This file wasn’t being used by anyone. It wasn’t exported either.

Reply:
what if we deleted it... 😄

Reply:
oh you did! wonderful.